pandas/tests/indexing/test_scalar.py

```diff
@@ -302,3 +302,40 @@ def test_iat_dont_wrap_object_datetimelike():
         assert result is ser2[1]
         assert isinstance(result, timedelta)
         assert not isinstance(result, Timedelta)
+
+
+def test_tuple_indexed_series_at_get():
+    # GH 26989
+    # Series.at works with index of tuples
+    series = Series([1, 2], index=[(1, 2), (3, 4)])
+    assert series.index.nlevels == 1
+    assert series.at[1, 2] == 1
+
+
+def test_tuple_indexed_series_at_set():
+    # GH 26989
+    # Series.at works with index of tuples
+    series = Series([1, 2], index=[(1, 2), (3, 4)])
+    series.at[1, 2] = 3
+    assert series.index.nlevels == 1
+    assert series.at[1, 2] == 3
+
+
+def test_multiindex_series_at_get():
+    # GH 26989
+    # Series.at works with MultiIndex
+    series = Series([1, 2], index=[[1, 2], [3, 4]])
+    assert series.index.nlevels == 2
+    assert series.at[1, 3] == 1
+    assert series.loc[1, 3] == 1
+
```

Comment:
Can you add a test for not providing all MultiIndex levels? (so eg `series.at[1]`). 
And also a test with a duplicated label?

Reply:
And maybe also an equivalent test for a DataFrame (it seems to work, but would be good to ensure we also have an explicit test for that)

Reply:
Sure - I have it mostly done, but ran into a question. For the series

    series = pd.Series([1, 2, 3], index=[[1, 2, 2], [3, 4, 4]])

i.e.

````
1  3    1
2  4    2
   4    3
dtype: int64
````

what should be the output of series.at[1, 3]? I was expecting to just get the scalar 1, but currently I'm getting

````
1  3    1
dtype: int64
````

After thinking about this - perhaps what we have currently is the desired output. My reason for this is that the shape doesn't change depending on the input - (1, 3) vs (2, 4).

Reply:
IMO, that should be a scalar (but that is also the reason that I asked to test it, to be sure this is covered ànd to discuss the semantics we want)